go.mod

```diff
@@ -2,21 +2,16 @@ module github.com/grafana/grafana
 
 go 1.23.1
 
-// Override Prometheus version because Prometheus v2.X is tagged as v0.X for Go modules purposes and Go assumes
-// that v1.Y is higher than v0.X, so when we resolve dependencies if any dependency imports v1.Y we'd
-// import that instead of v0.X even though v0.X is newer.
-replace github.com/prometheus/prometheus => github.com/prometheus/prometheus v0.52.0
-
```

Comment:
@spinillos Please see this. I commented about this in your PR https://github.com/grafana/grafana/pull/98742